packages/wallet/frontend/src/components/dialogs/CreatePaymentPointerDialog.tsx

```diff
@@ -119,6 +120,10 @@ export const CreatePaymentPointerDialog = ({
                         )}
                       />
                     </div>
+                    <Checkbox
+                      label="I want to use this Payment Pointer for Web Monetization"
+                      {...createPaymentPointerForm.register('isWM')}
+                    ></Checkbox>
```

Comment:
And also let's place the checkbox below the public name field.
```suggestion
                    <Checkbox
                      label="I want to use this Payment Pointer for Web Monetization"
                      {...createPaymentPointerForm.register('isWM')}
                    />
```